src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -384,6 +384,8 @@ public function fixTemplates(
 
         $methods = [];
         $pos = 0;
+        $templateRegistryDefinition = null;
+        $templateRegistryId = null;
         foreach ($definition->getMethodCalls() as [$method, $args]) {
```

Comment:
I don"t understand a lot this method.
Does it require some `NEXT_MAJOR` comment ? setTemplates and setTemplate will still be supported ?

Reply:
I just finish tag functionality. Now I will recatoring it and add documentation for it. Then changes will be describe. `NEXT_MAJOR` is missing but this is `WIP`.